downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -208,3 +208,35 @@ public function main() {
         string|crypto:PublicKey certFile?;
     |};
     ```
+
+#### `graphql` package
```

Comment:
This is not a backward-incompatible change, right? Currently listed under that section.

Reply:
Mistake. Fixed with 544bef54d20593b423108a1d5987f238f98ac864